modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/multiselect/MultiSelectFragment.kt

```diff
@@ -86,6 +90,7 @@ class MultiSelectFragment : BaseFragment(), MultiSelectTouchCallback.ItemTouchHe
                 multiSelectActions.add(multiSelectEpisodesHelper.maxToolbarIcons + 1, overflowTitle)
 
                 items = multiSelectActions.toList()
+                adapter.shouldShowRemoveListeningHistory = shouldShowRemoveListeningHistory
```

Comment:
Why setting the property here is needed?

Reply:
The `settings.multiSelectItemsObservable` triggers again after the adapter was created with a different value, so this was not hiding the row in the edit page